### ISSUE TITLE
[SIG-IRList] CfP LocWeb 2022 @ WWW2022 - 12th International Workshop on Location and the Web

### ISSUE THREAD

**Issue by karan96** [fani-lab/OpeNTF#93]
Prepare Timeline

**Comment by karan96** [comment]
@hosseinfani Closing this issue as it is not required anymore.

**Comment by hosseinfani** [comment]
@karan96 You were supposed to prepare a submission for this workshop. This was your first due date for a submission that you missed. This means that you're clearly behind your thesis schedule!

**Comment by karan96** [comment]
@hosseinfani I think we should really focus on the positives here. We are behind the schedule "we" developed for ourselves. As I have said, we should always make our deadlines more tight so that we are kept in check at all times and that we can not take any time for granted and be productive instead. The university does not care about when we present our thesis. Not trying to compare, nor looking down on anyone, but I have attended a lot of thesis presentations which either do not have a novel approach or are just a minor improvement on the previous idea, gladly we are not trying to be one of those. If I just wanted to do anything and be done with it, I would have selected a easy way out, but I did not. I set up those deadlines because they provide me the necessary push I need to work, and which is working well so far for me. I need to be keep myself in check by not wasting any time and being productive and hence the deadlines. But I really do not believe, we should have a negative mindset while doing so. Just my own thoughts, I totally respect your opinion and viewpoint as well.
-KS
